frontend/src/components/VisaStatusOverview.js

```diff
@@ -0,0 +1,265 @@
+
+const OveriviewHeaderBar = () => {
+    const [t] = useTranslation();
+    return (
+        <Row align="middle" className="ovreview-content-row" gutter={16}>
+            <Col xs={{ span: 0 }} md={{ span: 3 }}>
+                <strong>{t("Location")}</strong>
+            </Col>
+            <Col xs={{ span: 0 }} md={{ span: 4 }}>
+                <strong>{t("overviewEarliestDate")}</strong>
+            </Col>
+            <Col xs={{ span: 0 }} md={{ span: 4 }}>
+                <strong>{t("overviewLatestDate")}</strong>
+            </Col>
+            <Col xs={{ span: 0 }} md={{ span: 8 }}>
+                <strong>{t("overviewNewestFetch")}</strong>
+            </Col>
+            <Col xs={{ span: 0 }} md={{ span: 5 }}>
+                <strong>{t("overviewActions")}</strong>
+            </Col>
+        </Row>
+    );
+};
+
+const OverviewContentCard = ({ visaType, embassyCode, earliestDate, latestDate }) => {
+    const [t] = useTranslation();
+
+    return (
+        <Card
+            title={
+                <Row justify="space-between">
+                    <Col span={10}>{t(embassyCode)}</Col>
+                    <Space direction="horizontal">
+                        <Col span={3}>
+                            <TooltipBox title={t("overviewEmailIcon")}>
+                                <Button icon={<MailOutlined />} shape="circle" onClick={() => {}} />
+                            </TooltipBox>
+                        </Col>
+                        <Col span={3}>
+                            <TooltipBox title={t("overviewQQIcon")}>
+                                <Button icon={<QqOutlined />} shape="circle" onClick={() => {}} />
+                            </TooltipBox>
+                        </Col>
+                        <Col span={3}>
+                            <TooltipBox title={t("overviewAddtionalIcon")}>
+                                <Button icon={<EllipsisOutlined rotate={90} />} shape="circle" onClick={() => {}} />
+                            </TooltipBox>
+                        </Col>
+                    </Space>
+                </Row>
+            }
+        >
+            <Row>
+                <Col span={9}>
+                    <strong>{t("overviewEarliestDate")}: </strong>
+                </Col>
+                <Col span={15}>{earliestDate.join("/")}</Col>
+                <Col span={9}>
+                    <strong>{t("overviewLatestDate")}: </strong>
+                </Col>
+                <Col span={15}>{latestDate.join("/")}</Col>
+                <Col span={9}>
+                    <strong>{t("overviewNewestFetch")}: </strong>
+                </Col>
+                <Col span={15}>
+                    <OverviewNewest visaType={visaType} embassyCode={embassyCode} />
+                </Col>
+            </Row>
+        </Card>
+    );
+};
+OverviewContentCard.propTypes = { ...overviewPropTypes.overview };
+
+const OverviewChart = ({ visaType, embassyCode }) => {
```

Comment:
We should write this in another js file, as now the charts and overview list are placed in the same level and switched with tabs